superset/migrations/versions/e201cd43a98f_.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
+
+Revision ID: e201cd43a98f
+Revises: 3ba5164aa5dc
+Create Date: 2018-12-13 18:25:56.553826
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'e201cd43a98f'
+down_revision = '3ba5164aa5dc'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
```

Comment:
A migration for the existing entries needs to be added

Reply:
From my understanding this adds a column to the existing entries as well